flutter/lib/src/flutter_enricher_event_processor.dart

```diff
@@ -130,8 +129,18 @@ class FlutterEnricherEventProcessor extends EventProcessor {
     final tempDebugBrightnessOverride = debugBrightnessOverride;
     final initialLifecycleState = _window?.initialLifecycleState;
     final defaultRouteName = _window?.defaultRouteName;
+    final mode = kDebugMode
+        ? 'debug'
+        : kReleaseMode
+            ? 'release'
+            : 'profile';
```

Comment:
```suggestion
    final mode = _checker.isDebugMode()
        ? 'debug'
        : _checker.isReleaseMode()
            ? 'release'
            : 'profile';
```

So it can be tested.

Reply:
Did this but moved it to the Dart enricher.